tests/e2e/testcases/network/clean/suite.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package network
+
+import (
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("[Local Network] Clean", ginkgo.Ordered, func() {
+	ginkgo.It("can clean a started network", func() {
+		out := commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+
```

Comment:
do we want to add check for network status here to ensure that nothing is running? Feel free to ignore if you feel its redundant because of later tests